src/pcapi/tasks/decorator.py

```diff
@@ -31,6 +32,9 @@ def delay(payload: payload_in_kwargs):
                 f(payload)
                 return
 
+            if issubclass(payload.__class__, BaseModel):
```

Comment:
(détail)

Je pense qu'on devrait utiliser `isinstance()` (qui fonctionne aussi avec les sous-classes), plus simple :

```suggestion
            if isinstance(payload, pydantic.BaseModel):
```